pages/parent-enrollment.tsx

```diff
@@ -0,0 +1,76 @@
+import Wrapper from "@components/SDCWrapper";
+import { Flex, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseButton } from "@components/CloseButton";
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+    const [selectedChild, setSelectedChild] = useState<string>("");
+
+    return (
+        <Wrapper>
+            <Flex justifyContent="flex-end">
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/"></CloseButton>
+            </Flex>
+            <Center>
+                <Text align="center" mt="15px" fontWeight="700" fontSize="36px">
+                    Program Registration
+                </Text>
+            </Center>
+            <Center>
+                <Text pb="55px" align="center" mt="30px">
+                    Who would you like to register for{" "}
+                    <b>
+                        Building Bridges with Music - <br />
+                        Singing Monkeys (Ages 9 and under)?{" "}
+                    </b>
+                    (select one)
+                </Text>
+            </Center>
+
+            <Center>
+                <VStack spacing={5}>
+                    {children.map((childName) => {
+                        return (
```

Comment:
small stylistic comment but since the function passed into `map` doesn't do anything but return something, you don't need to do 
```
(childName) => { 
    return (
        <Button>...</Button>
    )
 }
```
but can rather just do 
```
(childName) => (
    <Button>...</Button>
)
```
i.e. omit the `return` . you don't have to change this necessarily but just an FYI